app/views/analyses/new/_phenotype_data_file_help.html.haml

```diff
@@ -0,0 +1,16 @@
+.alert.alert-info
+  Phenotype data needs to be provided in a CSV file formatted as shown by a <b>
+  #{link_to "downloadable template", new_analyses_data_file_path(data_type: :gwas_phenotype)}</b>:
+  %br
+  %br
+  %ul
+    %li Values in the CSV file must be comma-delimited. No spaces are allowed.
```

Comment:
No, I meant spaces in trait names, because they cannot be passed to GWASSER.R if they contain spaces. I updated help.